### ISSUE TITLE
High cpu usage

### ISSUE THREAD

**Issue by moerkey** [user-none/KDocker#64]
Hello,

after a resume from sleep mode the KDocker process creates a high cpu usage (100%). With every sleep there will be a further process.

I am using
- Ubuntu 18.04
- KDocker 5.3

Please let me know if I can support you with further information like debug logs, etc.

Cheers
Mark

**Comment by Daxx** [comment]
Were you successfully using other KDocker versions, before 5.3, on Ubuntu 18.04?

**Comment by moerkey** [comment]
Version 5.2 was also affected. I dont know if sth below too.

**Comment by Daxx** [comment]
I'm KDE so I wouldn't be able to reproduce, conveniently; perhaps someone else could.

If it's not too disruptive, could you arrange to undock just one of your docked apps each time before suspending and hope to find a culprit?
Do you get high CPU every time?

I've had problems with sleep in the past (13.04, I think) and I was told that the process would never be totally reliable because "interrupts" or something. Or maybe that was hibernate; one or the other, lol.

If no good result, we'd probably have to know all the apps (+versions) you had docked and steps to reproduce and even then, I doubt whether I'd be the one to help.

Please update here even if you're only thinking aloud. Thanks.

**Comment by moerkey** [comment]
It is mostly Firefox.

Can I undock and redock a process by commandline?

**Comment by Daxx** [comment]
> It is mostly Firefox. Can I undock and redock a process by commandline?

Not directly. Maybe this? :
`killall firefox && kdocker firefox`

or https://ubuntuforums.org/showthread.php?t=2380045

I think I'd want to check if firefox is unhappy about something on startup by starting once from the command line (without kdocker) and checking messages. Or if the FF cache is full or even HDD full, etc.

**Comment by moerkey** [comment]
Hm, I thought more about KDocker having a debug mode?

I already tried to kill firefox and restart it but I want to keep the session. So this is not a satisfying option.

Anyhow, I figured out that KDocker has a problem with Firefox or Thunderbird (or the other way around). 

JFYI I am running Trello, Firefox and Thunderbird with KDocker. The funny thing is that everything is running fine when I am starting Trello with KDocker at first and not FF or TH.

So it is kinda solved at first.
